### PR TITLE
Stop cloud server

### LINKED ISSUE #873: Enable users to stop cloud server

### PR DESCRIPTION
**Pull request by sukantoraymond**
closes #873

### CODE REVIEW

File: cmd/nodecmd/stop.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+
+	"github.com/spf13/cobra"
+)
+
+func newStopCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "stop [clusterName]",
+		Short: "(ALPHA Warning) Stop all nodes in a cluster",
+		Long: `(ALPHA Warning) This command is currently in experimental mode.
```
Comment (dasconnor) [inline cmd/nodecmd/stop.go:23]: You should add a note that stopping nodes may be insufficient to stop future charges and you may still be incur further costs

**Review by dasconnor — COMMENTED** [review]
One small comment then lgtm

File: cmd/nodecmd/stop.go [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+		delete(clusterConfig.Clusters, clusterName)
+	}
+	return app.WriteClusterConfigFile(&clusterConfig)
+}
+
+func removeDeletedNodeDirectory(clusterName string) error {
+	return os.RemoveAll(app.GetNodeInstanceDirPath(clusterName))
+}
+
+func removeClusterInventoryDir(clusterName string) error {
```
Comment (felipemadero) [inline cmd/nodecmd/stop.go:55]: this should be remove node from cluster inventory dir (similar in spirit to remove node from cluster config). can we add an issue for this? probably the inventory file should be regenerated based on the contents of the cluster config

File: cmd/nodecmd/stop.go [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+	clusterName := args[0]
+	var err error
+	clusterConfig := models.ClusterConfig{}
+	if app.ClusterConfigExists() {
+		clusterConfig, err = app.LoadClusterConfig()
+		if err != nil {
+			return err
+		}
+	}
+	clusterNodes := clusterConfig.Clusters[clusterName]
```
Comment (felipemadero) [inline cmd/nodecmd/stop.go:92]: doesn't it panic if there is no cluster config?

**Review by felipemadero — APPROVED** [review]
LGTM. Suggest wait also for connor approval for all this PRs

**Pull request merged**